source/_integrations/unifi.markdown

```diff
@@ -115,3 +115,9 @@ logger:
     homeassistant.components.device_tracker.unifi: debug
     homeassistant.components.switch.unifi: debug
 ```
+
+## FAQ
+
+### Understanding UniFi Naming (UniFi Controller vs. UniFi Network Application)
```

Comment:
```suggestion
### Understanding UniFi Naming (UniFi Network application is the UniFi Controller)
```

There are at least two instances of controller I couldn't add a comment on in MQTT and troubleshooting

Reply:
Yep, I only committed your suggestions. Going through it in depth now in my editor.